avionics/common/src/radio.cpp

```diff
@@ -48,14 +47,18 @@
 #endif
 
 namespace {
-constexpr uint64_t GND_STN_ADDR_MSB = 0x0013A200;
-constexpr uint64_t GND_STN_ADDR_LSB = 0x41678FC0;
+constexpr uint64_t kFlaregunAddrMsb = 0x0013A200;
```

Comment:
yep - all-caps are used by macros, which don't obey namespacing rules => prefix constants with `k` (following Google's naming style) to avoid this problem.